proposals/techical-alignment/proposal-specifications-graduation-to-jakarta.adoc

```diff
@@ -0,0 +1,26 @@
+# Innovations Graduate to Jakarta
+
+Under this proposal specifications ready for standardization are proposed as Jakarta specifications via Creation Review defined by the Jakarta EE Specification Process (JESP).
+
+ - API is repackaged into the `jakarta` namespace as currently defined by the Jakarta EE Specification Committee
```

Comment:
This will be very disruptive for the end users which are our main target. You simply don't do anything that harms your product.